patch/5.10-pensando-add-rtc-cpld.patch

```diff
@@ -0,0 +1,260 @@
+From 6a93d037ec20252d0d0b78824683bfdc934e2411 Mon Sep 17 00:00:00 2001
+From: Mushtaq Khan <mushkhan@pensando.io>
+Date: Tue, 4 Apr 2023 18:29:03 +0530
+Subject: [PATCH] rtc-elba-cpld-support
+
+---
+ arch/arm64/boot/dts/pensando/elba-asic-common.dtsi |  10 +-
+ arch/arm64/configs/elba_defconfig                  |   1 +
+ drivers/mfd/pensando-elbasr.c                      |   4 +
+ drivers/rtc/Kconfig                                |   9 ++
+ drivers/rtc/Makefile                               |   1 +
+ drivers/rtc/rtc-elbacpld.c                         | 156 +++++++++++++++++++++
+ 6 files changed, 176 insertions(+), 5 deletions(-)
+ create mode 100644 drivers/rtc/rtc-elbacpld.c
+
+diff --git a/arch/arm64/boot/dts/pensando/elba-asic-common.dtsi b/arch/arm64/boot/dts/pensando/elba-asic-common.dtsi
+index d8ab370..b1ce2c0 100644
+--- a/arch/arm64/boot/dts/pensando/elba-asic-common.dtsi
++++ b/arch/arm64/boot/dts/pensando/elba-asic-common.dtsi
+@@ -62,11 +62,6 @@
+ 		reg = <0x4c>;
+ 	};
+ 
+-	rtc@51 {
```

Comment:
This is will affect the Ortano cards running Sonic(RTC won't work on Ortano if we remove this change)..We need to probably need to create new dts for Sonic on elba with lipari. and add the below entry there.

Reply:
Ok, was not aware Sonic is supported on Ortano cards too. Let's discuss will schedule a meeting if we have to create new DTS file or if both rtc entries exist what will be the behavior?

Reply:
addressed by adding new dts  file for Lipari.